src/patternfly/components/EmptyState/empty-state.scss

```diff
@@ -64,7 +80,7 @@
 // RedHat Font overrides
 @include pf-m-redhat-font {
   .pf-c-empty-state {
-    > .pf-c-title {
+    > .pf-c-title:not(.pf-m-4xl) {
```

Comment:
The original selector here actually might be problematic. Since we don't set the `font-size` for the empty state title in the component's css, and instead use the title component's modifier to set that, users can modify the `font-size` if they wish by using a different title size modifier. If they've opted into the new font, we're overriding that behavior and forcing the font-size to `--pf-global--FontSize--xl` regardless the title modifier.

The intention here was just to change the default empty state title size from `lg` to `xl` when opting in to the new font without needing a markup change (modifying the title component's modifier). I think we could change this from `> .pf-c-title` to `> .pf-c-title.pf-m-lg` and consider that change fixing a bug, then we wouldn't have a need to do anything special for the xl variation title. wdyt?